cmd/Dockerfile

```diff
@@ -25,12 +25,15 @@ RUN --mount=type=cache,sharing=locked,target=/go/pkg/mod \
 
 FROM build-env as build-production
 WORKDIR /app
+
 ARG GOARCH=amd64
+ARG TAGS="relic,netgo"
```

Comment:
can you add a comment here explaining that `TAGS` is meant to be override if you don't want to build with `netgo` tag